packages/web/src/pages/oauth-login-page/hooks.ts

```diff
@@ -470,10 +473,15 @@ export const useOAuthSetup = ({
     } else if (scope === 'write_once') {
       // Note: Tx = 'connect_dashboard_wallet' since that's the only option available right now for write_once scope
       if ((tx as WriteOnceTx) === 'connect_dashboard_wallet') {
+        const sendWalletSignaturePromptToast = () => {
+          toast(messages.approveTxToConnectProfile, 1000 * 60 * 60 * 24)
```

Comment:
nit: A quick displaying what these numbers are doing could be useful for a quick glance. I assume this is 24hrs?
```suggestion
          toast(messages.approveTxToConnectProfile, 1000 * 60 * 60 * 24) // 24 hrs
```